src/test/java/org/openstreetmap/atlas/checks/validation/points/ConnectivityCheckTest.java

```diff
@@ -55,13 +55,12 @@ public void invalidConnectedNodesTest()
         this.verifier.globallyVerify(flags -> Assert.assertEquals(1, flags.size()));
     }
 
-    @Test
-    public void invalidDisconnectedEdgeCrossingTest()
-    {
-        this.verifier.actual(this.setup.invalidDisconnectedEdgeCrossingAtlas(),
-                new ConnectivityCheck(ConfigurationResolver.emptyConfiguration()));
-        this.verifier.globallyVerify(flags -> Assert.assertEquals(1, flags.size()));
-    }
+    /*
```

Comment:
I think we should probably remove this (remove commented section) to simplify and clear up confusion. If we leave it in there people may be confused as to why they are there.